src/DataCollector/ViewCollector.php

```diff
@@ -58,35 +58,9 @@ public function getWidgets()
      */
     public function addView(View $view)
     {
-        $name = $view->getName();
-        $path = $view->getPath();
-        $data = $view->getData();
-        $shortPath = '';
-        $type = '';
+        list($name, $type, $data, $path) = $this->getInertiaView($view->getName(), $view->getData(), $view->getPath());
```

Comment:
This title seems a bit odd if it's not for Inertia only?

Reply:
Yes, the method try inertia view, opposite case, fallback the same args
I'm using Google Translate and I didn't know what name to give it, plus I reused it from the ajax calls